astronomy/trappist_gravity_model.proto.txt

```diff
@@ -101,7 +101,7 @@ gravity_model {
     mass                 : "0.331 M🜨"
     reference_instant    : "JD2450000.000000000"
     mean_radius          : "0.773 R🜨"
-    axis_right_ascension : "90 deg"
+    axis_right_ascension : "-90 deg"
```

Comment:
Use these right ascensions:
`-89.9408, -89.9077, -89.9829, -90.0387, -89.9999, -89.9582, -89.9896, -89.8993` drawn from a normal distribution with μ=-90, σ=0.05, 
and these declinations:
`0.0497342, 0.0561637, 0.0787536, 0.0106394, -0.0949119, 0.0404001, 0.0367975, 0.0130609` drawn from a normal distribution with μ=0, σ=0.05, so that the axes are not embarrassingly parallel.

Reply:
Done.